From 1b044f64cc34dbff5128afe56598ce24990d21f9 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Sun, 14 Jan 2018 13:36:51 +0200
Subject: [PATCH 27/38] ARM: dts: imx7d: sbc-imx7: enable PCIe

Enable PCIe peripheral. Use ian internal clock input as
a Reference clock for the PCIe PHY.

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm/boot/dts/nxp/imx/imx7d-sbc-imx7.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/nxp/imx/imx7d-sbc-imx7.dts b/arch/arm/boot/dts/nxp/imx/imx7d-sbc-imx7.dts
index ba5ca7cc396a..d66f36f366cb 100644
--- a/arch/arm/boot/dts/nxp/imx/imx7d-sbc-imx7.dts
+++ b/arch/arm/boot/dts/nxp/imx/imx7d-sbc-imx7.dts
@@ -18,6 +18,7 @@
 
 	aliases {
 		lcdif = &lcdif;
+		pcie = &pcie;
 	};
 
 	backlight {
@@ -133,6 +134,12 @@
 	};
 };
 
+&pcie {
+	reset-gpio = <&pca9555_sb 1 GPIO_ACTIVE_LOW>;
+	ext_osc = <0>;
+	status = "okay";
+};
+
 &pwm2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_pwm2>;
-- 
2.20.1

